docs/development_suite/deploy/pipeline-based/configure-gitlab-runners.md

```diff
@@ -0,0 +1,166 @@
+---
+id: configure-gitlab-runners
+title: Deploy with GitLab Runners
+sidebar_label: Deploy with GitLab Runners
+---
+
+Mia-Platform Console can be configured to deploy your Project using GitLab Runners.
+
+To deploy your projects using GitLab Runners, you must add the GitLab Runners Provider in the Company Overview page, selecting the CI/CD Tool option, and then follow the [standard Provider configuration](/console/company-configuration/providers/configure-provider.mdx).
+
+![Provider](./img/gitlab-runners-provider.png)
+
+## Workflow
+
+In your Git working directory, create a new file called `.gitlab-ci.yml`.
```

Comment:
Of what? It may be better to specify that the context is the project repository?

Reply:
Oh yeah of course 😄